### ISSUE TITLE
Printing not showing in the console

### ISSUE THREAD

**Issue by JonasVautherin** [vxgmichel/aioconsole#40]
I guess that's a limitation, but I just want to make sure of it.

I am running an asyncio task that sometimes prints something (say a "progress status"). Ideally, I'd like to run it from `apython`, and have the prints show when they arrive. But they don't show directly (until I hit "enter" where they suddenly all appear).

Is that expected behavior, or is there a way to have a status showing spontaneously in the console?

And thanks for the project, I love aioconsole for prototyping and playing with asyncio!

**Comment by vxgmichel** [comment]
Hi @JonasVautherin, thanks for your report!

I haven't been able to reproduce the problem you describe using either aioconsole version 0.1.10 or 0.1.11.dev0 with python 3.7 on linux (with or without the `--no-readline` option). 

Here's my test program:
```python
import asyncio
from itertools import count

async def main():
    for x in count():
        print(x)
        await asyncio.sleep(1)

if __name__ == '__main__':
    asyncio.run(main())
```

Running with:
```bash
$ apython test.py # Or
$ apython --no-readline test.py
```
Could you provide more information about your environment (OS, python version, aioconsole version)?

> And thanks for the project, I love aioconsole for prototyping and playing with asyncio!

Thanks, your words are much appreciated :wink:

**Comment by vxgmichel** [comment]
Also keep in mind that the `print` function in the apython namespace is different from `builtins.print`. By default, the apython `print` [uses the console as `file` argument](https://github.com/vxgmichel/aioconsole/blob/47d80efae262a9248ba86ecac5b7a5032d83bf40/aioconsole/code.py#L64). This allows `print` to work properly in remote sessions. That's also the reason why `apython test.py` was working fine: `test.py` prints to `stdout` while apython is using `stderr`.

**Comment by vxgmichel** [comment]
The fix is now available - version [0.1.11](https://github.com/vxgmichel/aioconsole/releases/tag/v0.1.11), available on pypi :)

**Comment by JonasVautherin** [comment]
Wow, that was quick! And it's also already available on the Arch AUR :D. Awesome, thanks!
